### PR TITLE
Fix / Cleeng PayPal invalid url error (clean)

### PR DESCRIPTION
**Pull request by ChristiaanScheermeijer**
## Description

See #492 for more information. This PR simply fixes the Cleeng PayPal error but keeps the flow intact.

### CODE REVIEW

File: packages/ui-react/src/utils/location.ts [hunk 1]
```diff
@@ -12,6 +12,6 @@ export const modalURLFromLocation = (location: Location, u: keyof AccountModals
 };
 
 // Create a full modal url including hostname, mostly for external use (e.g paypal successUrl)
-export const modalURLFromWindowLocation = (u: keyof AccountModals, queryParams?: QueryParamsArg) => {
+export const modalURLFromWindowLocation = (u: keyof AccountModals | null, queryParams?: QueryParamsArg) => {
```
Comment (AntonLantukh) [inline packages/ui-react/src/utils/location.ts]: Is it still modal URL when the `u` param is not defined? Or should we have a separate function for this purpose?

Comment (ChristiaanScheermeijer) [reply packages/ui-react/src/utils/location.ts]: The goal of explicitly passing `null` to this function is to clear the modal parameter from the URL. We could create a separate util for this, but that would essentially do the same:

```tsx
export const closeModalURLFromWindowLocation = (queryParams?: QueryParamsArg) => {
  return createURL(window.location.href, { ...queryParams, u: null });
}
```

Alternatively, we could create a `createURLFromWindowLocation` util (following the convention) and empty the `u` param from the callee.

```tsx
const successUrlPaypal = offerType === 'svod' ? waitingUrl : createURLFromWindowLocation({ u: null });
```

**Review by langemike — APPROVED** [review]
Seems like a solid approach to me

File: packages/ui-react/src/utils/location.ts (hunk 1 above)
Comment (royschut) [reply packages/ui-react/src/utils/location.ts:15]: Or you can use the `createURLFromLocation` util (we do the same in the AccountModal useEffect):
```javascript
const successUrlPaypal = offerType === 'svod' ? waitingUrl : createURLFromLocation(location, { u: null });
```

Comment (royschut) [reply packages/ui-react/src/utils/location.ts]: Or you could use `createURL(window.location.href, { u: null });` from Checkout.

Comment (AntonLantukh) [reply packages/ui-react/src/utils/location.ts]: `createURL(window.location.href, { u: null })` looks good. We can have a small function for this. It is just a bit confusing to me now that we build a modal url without any modal.

Comment (ChristiaanScheermeijer) [reply packages/ui-react/src/utils/location.ts]: Updated

**Review by AntonLantukh — APPROVED** [review]

**Comment by ChristiaanScheermeijer** [comment]
@AntonLantukh @dbudzins can we merge this PR? 😄

**Pull request merged**